src/browser/api/application.js

```diff
@@ -657,7 +658,13 @@ function run(identity, mainWindowOpts, userAppConfigArgs) {
 
         coreState.removeApp(app.id);
 
-        if (!app._options._runtimeAuthDialog && !runtimeIsClosing && coreState.shouldCloseRuntime()) {
+        const shouldCloseRuntime =
+            app._options._type !== ERROR_TYPE_RENDERER_CRASH &&
```

Comment:
Are we still handling the shutdown after this somewhere? What was the behavior before that this changes?

Reply:
This simply prevents auto closing the runtime when you click "OK" button in the dialog. It's left up to the user to manually close apps with renderer crashes.